src/gui/controller/CreateCaseController.java

```diff
@@ -0,0 +1,46 @@
+package gui.controller;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.TabPane;
+
+public class CreateCaseController extends Controller {
+
+  /**
+   * Tabpane holding the different stages in case-creation
+   */
+  @FXML
+  private TabPane caseTabPane;
+
+  /**
+   * Initializes the controller class.
+   */
+  @Override
+  public void initialize(URL url, ResourceBundle rb) {
+    // TODO
+  }
+
+  /**
+   * Go to next tab in the process
+   *
+   * @param event
+   */
+  @FXML
+  private void nextButtonClicked(ActionEvent event) {
```

Comment:
We should validate and save the given data before moving to the next step?

Reply:
Was planning on adding it, but we haven't decided upon how we will save it yet